app/controllers/api/v2/dlmlocks_controller.rb

```diff
@@ -97,6 +97,12 @@ def release
         process_lock_response @dlmlock.release!(@host)
       end
 
+      api :GET, '/dlmlocks/:id/events', N_('List all events for a given DLM lock')
+
+      def events
```

Comment:
I think, I'd move this to `DlmlocksDlmLockEventsController#index`. I think that makes permissions etc. easier.

See https://github.com/theforeman/foreman/blob/ae96cb70130968bdcc8b7a9a9af7edc7c6e0d9a8/app/controllers/api/v2/images_controller.rb for an example.